include/Support/Check.h

```diff
@@ -0,0 +1,20 @@
+//===- Check.h - Functions for checking test results-------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+//
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef OFFLOADTEST_SUPPORT_CHECK_H
+#define OFFLOADTEST_SUPPORT_CHECK_H
+
+#include "Pipeline.h"
+
+bool getResult(offloadtest::Result R);
```

Comment:
Can you add a doxygen comment describing what this does?

Since I think this is verifying the correctness of the result a better name might be something like `verifyResult`.

Reply:
I updated the name and added a comment